spring-core/src/main/java/org/springframework/cglib/core/ReflectUtils.java

```diff
@@ -260,7 +260,7 @@ private static Class getClass(String className, ClassLoader loader, String[] pac
 		while ((index = className.indexOf("[]", index) + 1) > 0) {
 			dimensions++;
 		}
-		StringBuffer brackets = new StringBuffer(className.length() - dimensions);
+		StringBuilder brackets = new StringBuilder(className.length() - dimensions);
```

Comment:
In light of the following policy, please revert this change.

----

Please refrain from modifying classes under `org.springframework.asm`, `org.springframework.cglib`, and `org.springframework.objenesis`. Those include repackaged forks of the third-party libraries ASM, CGLIB, and Objenesis. Any refactoring to those classes should take place upstream in the originating repository. The Spring Framework will then pick up the changes when syncing with official updates of the forked third-party libraries.

Reply:
@sbrannen done.